### ISSUE TITLE
Serious bugs

### ISSUE THREAD

**Issue by Jackberg** [facebookresearch/ParlAI#681]
Tried to pull the up-to-date code, but I met serious bugs, and the program won't even run now.

First is the argument problem, it seems the program is confused by argument abbreviations, e.g. `-hs` here:
```
usage: train_model.py [-h] [-t TASK] [--download-path DOWNLOAD_PATH]
                      [-dt {train,train:stream,train:ordered,train:ordered:stream,train:stream:ordered,valid,valid:stream,test,test:stream}]
                      [-im IMAGE_MODE] [-nt NUMTHREADS] [-bs BATCHSIZE]
                      [-bsrt BATCH_SORT] [-clen CONTEXT_LENGTH]
                      [-incl INCLUDE_LABELS] [-dp DATAPATH] [-m MODEL]
                      [-mf MODEL_FILE] [--dict-class DICT_CLASS]
                      [-et EVALTASK] [--display-examples DISPLAY_EXAMPLES]
                      [-eps NUM_EPOCHS] [-ttim MAX_TRAIN_TIME]
                      [-ltim LOG_EVERY_N_SECS] [-vtim VALIDATION_EVERY_N_SECS]
                      [-stim SAVE_EVERY_N_SECS] [-sval SAVE_AFTER_VALID]
                      [-vme VALIDATION_MAX_EXS] [-vp VALIDATION_PATIENCE]
                      [-vmt VALIDATION_METRIC] [-vmm {max,min}]
                      [-vcut VALIDATION_CUTOFF] [-dbf DICT_BUILD_FIRST]
                      [-lfc LOAD_FROM_CHECKPOINT]
train_model.py: error: argument -h/--help: ignored explicit argument 's'
The program exited via sys.exit(). Exit status: 2
```
But even if I change it to full name `--hiddensize`, there are still `Uncaught exception`.

**Comment by Jackberg** [comment]
The `Uncaught exception` problem was caused by PyTorch version, so currently the problem only lies in the abbreviation of the argument name.

**Comment by alexholdenmiller** [comment]
Hi @Jackberg thanks for raising, I'll have this fixed by EOD

**Comment by alexholdenmiller** [comment]
For now please use the full-length arguments for any that raise an issue

**Comment by alexholdenmiller** [comment]
fix in

**Comment by Jackberg** [comment]
Yes, working now. But there's still a small bug: 
When I try `python examples/train_model.py -m seq2seq -h`, it won't show the help message for seq2seq model.

**Comment by alexholdenmiller** [comment]
fix up, thanks for flagging! #692
